packages/mobile/src/services/offline-downloader/workers/playCounterWorker.ts

```diff
@@ -14,4 +14,20 @@ const countPlay = async (payload: PlayCountWorkerPayload) => {
   store.dispatch(recordListen(trackId))
 }
 
+export const setPlayCounterWorker = async (
+  worker: Worker<PlayCountWorkerPayload>
+) => {
+  queue.stop()
+  queue.removeWorker(PLAY_COUNTER_WORKER)
+  queue.addWorker(worker)
+  await queue.start()
```

Comment:
not sure if this really needs to be await since nothing is "waiting" for it. but just an observation. looks good!

Reply:
yeah I had the same thought. Not sure if it matters